doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc

```diff
@@ -17,8 +17,10 @@ This pane lists the following process attributes which you can use to filter pro
 ... Click *Apply*.
 +
 
-* *Name*: Filter process instances based on process definition name.
-* *Start Date*: Filter process instances based on their creation date.
-* *Last update*: Filter process instances based on their last modified date.
-
-You can also use the *Advanced Filters* option to create custom filters in {CENTRAL}.
+* *Name*: Filter process instances by definition names.
+* *Definition Id*: Filter process instances by process definition Ids.
```

Comment:
Normally, I'd say IDs not Ids. Search in doc. But I see it's Id in the UI. Ugh...

Reply:
I changed the instances that were not UI replicas